doc/spire_agent.md

```diff
@@ -44,7 +44,7 @@ The following configuration options are available to configure a plugin:
 | --------------- | ---------------------------------------- |
 | plugin_cmd      | Path to the plugin implementation binary (optional, not needed for built-ins) |
 | plugin_checksum | An optional sha256 of the plugin binary  (optional, not needed for built-ins) |
-| enabled         | Enable or disable the plugin             |
```

Comment:
nitpick: to be consistent with structure of other values, maybe:
`| enabled             | Enable or disable the plugin (enabled by default)    |`